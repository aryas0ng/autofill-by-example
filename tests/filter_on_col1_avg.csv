1,0,2
1,4,2
4,3,3
6,8,7.66666666666667
6,10,7.66666666666667
6,5,
19,4,
19,4,
19,21,
19,21,
9,10,
7,7,
7,4,
3,3,
3,3,
3,16,
1,16,
1,27,
1,8,
3,3,
3,3,
8,11,
8,4,