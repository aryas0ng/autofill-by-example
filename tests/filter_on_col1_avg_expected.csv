1,0,2
1,4,2
4,3,3
6,8,7.66666666666667
6,10,7.66666666666667
6,5,7.66666666666667
19,4,12.5
19,4,12.5
19,21,12.5
19,21,12.5
9,10,10
7,7,5.5
7,4,5.5
3,3,7.33333333333333
3,3,7.33333333333333
3,16,7.33333333333333
1,16,17
1,27,17
1,8,17
3,3,3
3,3,3
8,11,7.5
8,4,7.5